Cargo.lock

```diff
@@ -2053,15 +2053,6 @@ dependencies = [
  "hex",
 ]
 
-[[package]]
-name = "demo-ncompose"
-version = "0.1.0"
-dependencies = [
- "gear-wasm-builder",
- "gstd",
- "hex",
-]
```

Comment:
Some examples seem to be accidentally removed.

Reply:
yeah just fount it is pretty buggy, considering creating another issue for the examples

Reply:
https://github.com/gear-tech/gear/issues/3069